lib/ruby-mpd/parser.rb

```diff
@@ -26,6 +26,8 @@ def convert_command(command, *params)
           end
         when MPD::Song
           quotable_param param.file
+        when MPD::Playlist
```

Comment:
Playlists can be passed as params in command lists methods where a playlist name is required.